docs/arcaflow/workflows/expressions.md

```diff
@@ -260,6 +260,7 @@ Functions:
 | `toLower(string)`                                | string       | Returns the input with Unicode letters mapped to their lower case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `toUpper(string)`                                | string       | Returns the input with Unicode letters mapped to their upper case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `splitString(string, string)`                    | list[string] | Returns a list of the substrings which appear between instances of the specified separator; the separator instances are not included in the resulting list elements; adjacent occurrences of separator instances as well as instances appearing at the beginning or ending of the input will produce empty string list elements.<br>&nbsp; Param 1: The string to split.<br>&nbsp; Param 2: The separator.                                                                                     |
+| `readFile(string)`                    | string | Given a string containing a file path, returns the contents of the indicated file as a string.<br>&nbsp;                                                                                     |
```

Comment:
The documentation already isn't completely consistent, but I'm not thrilled about adding yet another pattern. The most common is "Returns a ...", which I think would fit nicely here. "Returns the contents of a named file path as a string", for example.

As I mentioned earlier, I'm curious what we can say about the file path: specifically, against what working directory by default. We could also say that we return the file contents as a utf-8 character string rather than, say, as a byte string.

Reply:
`UTF-8 character string` seems like it would be better in the `return type` column than in the description

Reply:
> but UTF-8 character string seems like it would be better in the return type column

Maybe. That seems to be assumed in most cases, or maybe doesn't matter so much since all internal codings of `string` generally match. In terms of reading byte streams from a file, however, there's some inherent ambiguity. But does it really make sense to use a different "return type" value here than in other functions that return strings? Or does that just get confusing... ??

Reply:
`toUpper()`, e.g., has a "return type" of `string` but mentions "Unicode" in its description.  With that as a basis, I would leave the return type for `readFile()` as `string`; however, it probably is worth mentioning in the description that the character encoding is UTF-8 and not ASCII.